valohai/internals/global_state_loader.py

```diff
@@ -102,6 +108,10 @@ def parse_overrides_from_cli(
         if isinstance(value, bool):
             # We need to fiddle booleans in a bit different way, since they are treated as flags per default
             parser.add_argument(f"--{name}", type=string_to_bool, nargs="?", const=True)
+        elif isinstance(value, list):
+            parser.add_argument(
+                f"--{name}", type=string_to_list_separated_by_separator()
+            )
```

Comment:
Can we YAGNI/keep-it-simple this to
```suggestion
            parser.add_argument(f"--{name}", type=lambda s: str(s).split(","))
```